docs/docs/guides/cli/how-to-use-lightdash-validate.mdx

```diff
@@ -0,0 +1,38 @@
+import AutomaticallyDeployChangesGithub from './../../snippets/github-secrets.mdx';
+
+# Lightdash Validate
+
+You can trigger a validation on a project from the CLI, this will allow you to block a release (using Github actions) 
+if a table/chart/dashboard is broken on a [preview environment](/guides/cli/how-to-use-lightdash-preview). 
+
+Preview environments will copy all spaces/charts/dashboards into your new preview environment. 
+This is only copied on preview creation, you can't sync the content afterwards. 
+
+## Usage 
+
+```
+# This will trigger a validation for the last preview project you create using `lightdash start-preview`
+lightdash validate 
+```
+
+or 
+
+```
+# Trigger validation for any project UUID 
+lightdash validate --project <project uuid> 
+``` 
+
+![CLI validate](./assets/cli-validate.png)
+
+
+## Configure Github actions  
```

Comment:
Can we add a bit here that references the docs on how to set up GitHub actions? I think there are docs where we talk about the templates? 🤔 

Or maybe we actually need to create a separate reference doc just for GitHub actions + all the actions you can do (I can create a separate issue for that). 
 

In the meantime though, maybe we can add something like this at the bottom of the section?:

```
To learn more about setting up GitHub actions for Lightdash, check out the docs here..
```